### PR TITLE
DEVDOCS-5045: [update] Remove non-token oauth scopes from token creation docs

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5045]

## What changed?
Specific OAuths are not needed

## Release notes draft

* 

## Anything else?

ping {names}

[DEVDOCS-5045]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5045?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/api-docs/getting-started/authentication.mdx [hunk 1]
```diff
@@ -158,7 +158,7 @@ The following table lists the APIs that use the `Authorization` header, along wi
 | [GraphQL Storefront API](/docs/storefront/graphql) | [Create a token](/docs/rest-authentication/tokens#create-a-token), Stencil context | [Create a Storefront query](/docs/graphql-storefront) | store | `Bearer {{TOKEN}}` |
 | [Payment processing](/docs/store-operations/payments) endpoint | [Create a payment access token](/docs/rest-payments/tokens#create-payment-access-token), `completeCheckout` with the [GraphQL Storefront API](/docs/storefront/cart-checkout/guide/graphql-storefront#handling-payments) | [Process a payment](/docs/rest-payments/processing#process-payment) | app or store | `PAT {{TOKEN}}` |
 
-To obtain a dynamic token, send a request to the REST endpoint listed in the **Obtain a JWT** column or consult the article listed in the **API Description** column. Use an API account with the appropriate [token creation OAuth scopes](/docs/start/authentication/api-accounts#token-creation-scopes).
+To obtain a dynamic token, send a request to the REST endpoint listed in the **Obtain a JWT** column or consult the article listed in the **API Description** column. Use an API account with a [token creation scope](/docs/start/authentication/api-accounts#token-creation-scopes).
```
Comment (slsriehl) [inline docs/api-docs/getting-started/authentication.mdx:161]:
```suggestion
To obtain a dynamic token, send a request to the REST endpoint listed in the **Obtain a JWT** column or consult the article listed in the **API Description** column. Use an API account with the appropriate [token creation scope](/docs/start/authentication/api-accounts#token-creation-scopes) for the kind of tokens you wish to generate.
```

File: docs/api-docs/payments/payments-api-overview.mdx [hunk 2]
```diff
@@ -13,7 +13,7 @@ Process payments by making a sequence of requests to the following two API endpo
 
 <Callout type="info">
 #### Required OAuth scopes
-Consult the [Payments API Reference](/docs/rest-payments) to determine which OAuth permission scopes your application's [API Account](/docs/start/authentication/api-accounts) needs.
+Consult the [Create Payment Access Token](/docs/rest-payments) endpoint to use the OAuth permission scope for your application's [API Account](/docs/start/authentication/api-accounts).
```
Comment (slsriehl) [inline docs/api-docs/payments/payments-api-overview.mdx:16]:
```suggestion
Add the [Create Payments scope](/docs/start/authentication/api-accounts#token-creation-scopes) to the [store-level or app-level API account](/docs/start/authentication/api-accounts) you use when you [Create Payment Access Tokens](/docs/rest-payments/tokens#create-payment-access-token).
```

File: docs/api-docs/storefront/graphql/graphql-api-overview.mdx [hunk 3]
```diff
@@ -158,7 +158,7 @@ This auto-generated token has an expiry period of 24-48 hours and will periodica
 
 ### Customer impersonation tokens
 
-It's also possible to generate tokens for use in server-to-server or headless interactions with a trusted consumer using the [Create a customer impersonation token](/docs/rest-authentication/tokens#customer-impersonation-tokens) endpoint. Consult the endpoint's documentation to determine what OAuth scopes it requires.
+It's also possible to generate tokens for use in server-to-server or headless interactions with a trusted consumer using the [Create a customer impersonation token](/docs/rest-authentication/tokens/customer-impersonation-token) endpoint. Use the Storefront API Customer Impersonation Tokens [OAuth scope](/docs/rest-authentication/tokens/customer-impersonation-token#create-a-token) in the request.
```
Comment (slsriehl) [inline docs/api-docs/storefront/graphql/graphql-api-overview.mdx:161]:
```suggestion
You can use Customer Impersonation Tokens to authenticate requests to the GraphQL Storefront API in server-to-server and headless interactions. Add the [Storefront API Customer Impersonation Tokens scope](/docs/start/authentication/api-accounts#token-creation-scopes) to the [store-level or app-level API account](/docs/start/authentication/api-accounts) you use when you [Create a customer impersonation token](/docs/rest-authentication/tokens/customer-impersonation-token). 
```

File: docs/api-docs/storefront/graphql/graphql-api-overview.mdx [hunk 4]
```diff
@@ -102,7 +102,7 @@ curl 'https://{bigcommerce_storefront_domain}.com/graphql'\
 
 ### Creating a token
 
-Use the [Create a GraphQL Storefront API token](/docs/rest-authentication/tokens#create-a-token) REST endpoint to request JWT bearer tokens that authenticate requests to the GraphQL Storefront API.
+Use the [Create a storefront token](/docs/rest-authentication/tokens#create-a-token) REST endpoint to request JWT bearer tokens that authenticate requests to the GraphQL Storefront API. Use a [token creation scope](/docs/start/authentication/api-accounts#token-creation-scopes) for the [store-level or app-level API account](/docs/start/authentication/api-accounts) in the request.
```
Comment (slsriehl) [inline docs/api-docs/storefront/graphql/graphql-api-overview.mdx:105]:
```suggestion
Use the [Create a storefront token](/docs/rest-authentication/tokens#create-a-token) REST endpoint to request JWT bearer tokens that authenticate requests to the GraphQL Storefront API. Add a [token creation scope](/docs/start/authentication/api-accounts#token-creation-scopes) to the [store-level or app-level API account](/docs/start/authentication/api-accounts) you use to generate tokens.
```

Comment (bc-traciporter) [reply docs/api-docs/storefront/graphql/graphql-api-overview.mdx:105]: I had it like this before and changed it.

**Review by slsriehl — APPROVED** [review]

**Pull request merged**